### PR TITLE
[Mac] Make GetApplicationUrl tests resilient to any kind of system.

### PR DESCRIPTION
**Pull request by Therzok**

### CODE REVIEW

**Comment by Therzok** [comment]
Possibly should add some comments:
CF calls do not work if the extension is not `plist`. So using the `Info.plist.in` does not work.

File: main/tests/MacPlatform.Tests/CoreFoundationTests.cs [hunk 1]
```diff
@@ -56,19 +57,52 @@ public static void TestURL ()
 		[Test]
 		public static void TestApplicationUrls ()
 		{
-			string test = "/Applications/Xamarin Studio.app/Contents/Info.plist";
-			string[] results = MonoDevelop.MacInterop.CoreFoundation.GetApplicationUrls (test, MonoDevelop.MacInterop.CoreFoundation.LSRolesMask.All);
+			string dir = nameof (TestApplicationUrl);
```
Comment (slluis) [inline main/tests/MacPlatform.Tests/CoreFoundationTests.cs]: Shouldn't this be a temp dir?

Comment (Therzok) [inline main/tests/MacPlatform.Tests/CoreFoundationTests.cs]: That's an even better idea!

**Comment by Therzok** [comment]
PR updated

**Pull request merged**